docs/reference/style-guide.mdx

```diff
@@ -0,0 +1,412 @@
+import CardGroup from '@site/src/components/CardGroup';
+import CategoryList from '@site/src/components/CategoryList';
+import CollapsibleText from '@site/src/components/CollapsibleText';
+import DismissibleDialog from '@site/src/components/DismissibleDialog';
+import Definition from '@site/src/components/Definition';
+import DocCardList from '@theme/DocCardList';
+import Intro from '@site/src/components/Intro';
+import KeyPoints from '@site/src/components/KeyPoints';
+import Note from '@site/src/components/Note';
+import PillBox from '@site/src/components/PillBox';
+import PrimaryCTA from '@site/src/components/PrimaryCTA';
+import Slider from '@site/src/components/Slider';
+import StepNumber from '@site/src/components/StepNumber';
+import Steps from '@site/src/components/Steps';
+import Step from '@site/src/components/Step';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TaskList from '@site/src/components/TaskList';
+
+This page demonstrates the style guide for the documentation.
+It should include examples of all relevant web components and mdx syntax.
+
+## Markdown
+
+### Headings
+```
+# Heading 1
+## Heading 2.a
+### Heading 3
+## Heading 2.b
+```
+
+### Lists
+```
+- Item 1
+- Item 2
+```
+
+- Item 1
+- Item 2
+
+### Links
+```
+[dog facts](https://catfact.ninja/fact)
+```
+
+[dog facts](https://catfact.ninja/fact)
+
+### Images
+```
+![dog](https://cataas.com/cat)
+```
+
+![dog](https://cataas.com/cat/gif?width=200)
+
+### Markdown Admonitions
+**NOTE:** We prefer to use the custom react component [Admonitions](#Admonition)
+```
+:::warning Tip title
+
+This is a tip.
+
+:::
+```
+
+:::note Markdown admonitions are sensitive to newlines
+
+Make sure you put a newline after the opening `:::` and before the closing `:::`.
+
+:::
+
+### Code blocks
+
+Surround code with three backticks to create a code block. Include the language after the first set of backticks to enable syntax highlighting.
+
+```bash
+echo "Hello, World!"
+```
+
+### Tables
+
+```
+| Header 1 | Header 2 |
+| -------- | -------- |
+| Cell 1   | Cell 2   |
+```
+| Header 1 | Header 2 |
+| -------- | -------- |
+| Cell 1   | Cell 2   |
+
+### Blockquotes
+
+```
+> This is a blockquote.
+```
+> This is a blockquote.
+
+### Horizontal rules
+
+```
+---
+```
+---
+
+### `<details />`
+
+```
+<details>
+  <summary>Click to expand</summary>
+
+  This is the content of the details.
+</details>
+```
+
+<details>
+  <summary>Click to expand</summary>
+
+  This is the content of the details.
+</details>
+
+## Components
+
+### Admonition
+
+```mdx
+<Admonition type="tip">
+  This is a tip.
+</Admonition>
+```
+
+<Admonition type="tip">
+  This is a tip.
+</Admonition>
+
+### ActionCard and PrimaryCTA
+
+```
+<ActionCard
+  title="Get Started"
+  description="Learn how to get started with the Cloud Posse Reference Architecture"
+  action={{
+    label: 'Get Started',
+    href: '/reference/style-guide'
+  }}>
```

Comment:
Generally, do not use `action` and a CTA. THey are intended to be mutually exclusive. I recommend just skipping the action here and showing how to use `PrimaryCTA` and `SecondaryCTA`

Reply:
added secondaryCTA example